it/v4/endpoints/CreateUkPropertyPeriodSummaryControllerISpec.scala

```diff
@@ -0,0 +1,586 @@
+/*
```

Comment:
There's a `println` statement left in v2 of this iSpec. May as well delete it whilst we're here?